vault/core.go

```diff
@@ -2424,6 +2424,7 @@ func (c *Core) SetLogLevel(level log.Level) {
 // SetConfig sets core's config object to the newly provided config.
 func (c *Core) SetConfig(conf *server.Config) {
 	c.rawConfig.Store(conf)
+	c.logger.Debug("set config", "sanitized config", c.SanitizedConfig())
```

Comment:
This seems consistent with the rest of Vault, but I wonder if we instead print JSON-encoded (non-indented/minified) config would be easier to grok?

Reply:
Probably.  I didn't want to get derailed from my current work so this is the MVP :)

If you're so inclined, feel free to add a commit to this branch to make it look better.

Reply:
I would prefer that whatever we do, it's emitted on a single line, so it can be easily grepped.

Reply:
Updated :)